### DISCUSSION TITLE
Running "interactive-session-with-examples" gives the following error.

### DISCUSSION THREAD

**Discussion by 90PlusWorks** [OpenModelica/OpenModelica#11161] [Q&A]
The following example comes from: https://openmodelica.org/doc/OpenModelicaUsersGuide/1.21/introduction.html#interactive-session-with-examples

>>loadFile(getInstallationDirectoryPath() + "/share/doc/omc/testmodels/bubblesort.mo")
true
**Error occurred building AST
Syntax Error
[<interactive>:1:0-1:7:writable] Error: Parser error: Unexpected token near: loadFile (IDENT)**

Previous examples worked. I'm running the script in the OMShell on Windows.

**Comment by lucas-mior** [comment]
If you are following the documentation for the same openmodelica release, then you probably found a bug (either in the docs or in the bubblesort.mo file).

**Reply by casella** [reply]
@90PlusWorks can you just run getInstallationDirectoryPath() and report what output you get? It could be yet another issue with special characters in Windows paths.
